frontend/pages/create.tsx

```diff
@@ -107,12 +93,12 @@ export default function SpaceViewPage() {
     }
 
     if (
-      membership ===
-        memberOptions.findIndex((opt) => opt.id === "token_gated") &&
-      !tokenMembershipAddress
+      membershipAccess ===
+        memberAccessOptions.findIndex((opt) => opt.id === "gated") &&
```

Comment:
Then in places like this I would recommend: 
`memberAccessOptions.findIndex((opt) => opt.id === Access.GATED)`